assets/content/cookbook/Macros/compare-anon-structures.md

```diff
@@ -0,0 +1,131 @@
+# Compare Anonymous Structures and Objects
+
+## Why use a macro
+
+Sometimes it can be useful to compare two anonymous structures or objects
+ by their properties, even if they're not the same.
```

Comment:
We should clarify here that what we're after is structural equality, and that the objects are not the same physically. Otherwise the notion of "same" is confusing.